src/App.js

```diff
@@ -18,6 +20,11 @@ function App() {
           <Home allProducts={allProducts} changePage={setActivePage} />
         )}
       </header>
+
+       //prints the edit info form to the screen. This is temporary 
+       //for checking page.
+       <DisplayForm />
```

Comment:
I think, for development purposes, before Matthew's PR on page state is merged, you can do the same what is done on line 19 - 21 for Home component.  And Change useState value on line 14 to edit page. 

{activePage === App.pages.EDIT_PRODUCT&& (
          <DisplayForm />
        )}